aten/src/ATen/ParallelNativeTBB.cpp

```diff
@@ -6,6 +6,8 @@
 #include <mutex>
 
 #include "tbb/tbb.h"
+#define TBB_PREVIEW_GLOBAL_CONTROL 1
```

Comment:
The only thing that makes me a bit queasy here is that this is an experimental feature.

Reply:
well, yes, but this is also in official docs (https://software.intel.com/en-us/node/589744)
and that's the only setting I know that actually works

Reply:
the example in https://software.intel.com/en-us/node/589744 actually doesn't have "#define TBB_PREVIEW_GLOBAL_CONTROL 1" so I guess it might not be so experimental in the latest version

Reply:
also there's no mention of TBB_PREVIEW_GLOBAL_CONTROL in current intel/tbb repo